_data/projects/websurfx.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Websurfx
+desc: An open source alternative to searx which provides a modern-looking, lightning-fast, privacy-respecting, secure meta search engine written in Rust.
+site: https://github.com/neon-mmd/websurfx
+tags:
+- web-designing
+- foss
+- scraping
+- web
+- handlebars
+- css
+- actix-web
+- rust
+upforgrabs:
+  name: Up-for-Grabs
+  link: https://github.com/neon-mmd/websurfx/labels/Up-for-Grabs
```

Comment:
This label doesn't point to any active issue - I see you're using some different labels in that repository, so feel free to change this and the `name` value above to something else if you'd prefer...

Reply:
> This label doesn't point to any active issue - I see you're using some different labels in that repository, so feel free to change this and the `name` value above to something else if you'd prefer...